app/metrics/filesystem.go

```diff
@@ -69,33 +57,23 @@ func CollectFilesystem() ([]Metric, error) {
 			return nil, err
 		}
 
-		size := uint64(st.Blocks) * uint64(st.Bsize) / fsBlockSize
-		free := uint64(st.Bavail) * uint64(st.Bsize) / fsBlockSize
+		size := st.Blocks * uint64(st.Bsize) / fsBlockSize
```

Comment:
Is this working on all archs? we had an issue on arm that was solved by this:
https://github.com/qbee-io/qbee-agent/commit/1362ffb5281ceda58470438fac8ba39e60fa37b9

Reply:
I guess I became a victim of IDE suggestions. I'll restore and add a comment above, to make sure no one removes that in the future.

Reply:
For the record, it seems that all currently supported architecture use uint64 for those values:
- https://cs.opensource.google/go/go/+/refs/tags/go1.22.1:src/syscall/ztypes_linux_386.go;l=119
- https://cs.opensource.google/go/go/+/refs/tags/go1.22.1:src/syscall/ztypes_linux_amd64.go;l=119
- https://cs.opensource.google/go/go/+/refs/tags/go1.22.1:src/syscall/ztypes_linux_arm.go;l=120
- https://cs.opensource.google/go/go/+/refs/tags/go1.22.1:src/syscall/ztypes_linux_arm64.go

So it should work. I've also build packages and there weren't any issues.